autoware_adapi_v1_msgs/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ rosidl_generate_interfaces(${PROJECT_NAME}
   manual/msg/HazardLightsCommand.msg
   manual/msg/ManualControlMode.msg
   manual/msg/ManualControlModeStatus.msg
-  manual/msg/ManualOperatorStatus.msg
+  manual/msg/ManualOperatorHeartbeat.msg
```

Comment:
Would it be appropriate to include this change in this PR?

Reply:
It needed some other changes so I merged it into this PR. I updated the PR title and description so could you please check it again? The `steering_tire_velocity` additions have also been merged.